orchestra/orchestra_api.py

```diff
@@ -77,13 +78,13 @@ def create_orchestra_project(client,
     return project_id
 
 
-def get_project_information(project_id):
+def get_project_information(project_ids):
     data = {
-        'project_id': project_id
+        'project_ids': project_ids
     }
     response = _make_api_request('post', 'project_information',
                                  data=json.dumps(data))
-    return json.loads(response.text)
+    return json.loads(response.text, object_hook=convert_key_to_int)
```

Comment:
shouldn't this be done on the server side?

Reply:
It's done server-side, but since `project_information` is a `json_view`, contents are received in json which doesn't allow integer keys.